src/components/Notifications/NoticeBoard.tsx

```diff
@@ -14,29 +15,44 @@ export const NoticeBoard = () => {
   const { data, loading } = useTanStackQueryInstead(routes.getNotifications, {
     query: { offset: 0, event: "MESSAGE", medium_sent: "SYSTEM" },
   });
-
   let notices;
 
   if (data?.results.length) {
     notices = (
-      <div className="grid grid-cols-1 gap-6 sm:grid-cols-2 xl:grid-cols-3">
+      <div className="grid grid-cols-1 gap-6 sm:grid-cols-2 xl:grid-cols-3 mt-4">
         {data.results.map((item) => (
           <div
             key={`usr_${item.id}`}
-            className="overflow-hidden rounded shadow-md"
+            className="overflow-hidden rounded shadow-md my-1"
           >
-            <div className="px-6 py-4">
-              <div className="text-justify text-lg" id="notification-message">
-                {item.message}
-              </div>
-              <div className="text-md my-2 text-secondary-700">
-                {formatName(item.caused_by)} -{" "}
-                <span className="font-bold text-primary-700">
-                  {item.caused_by.user_type}
-                </span>
-              </div>
-              <div className="text-xs text-secondary-900">
-                {t("on")}: {formatDateTime(item.created_date)}
+            <div
+              className="text-justify text-lg flex mx-2 py-3 px-5"
+              id="notification-message"
+            >
+              <CareIcon
+                icon="l-facebook-messenger"
+                className="text-xl mr-6 mt-1"
+              />
+              <span className="font-mono text-md">{item.message} </span>
```

Comment:
i don't think mono font aligns with our designs.

Reply:
@rithviknishad if I use the same font that doesnt specify the message text explictlity

Reply:
@vinutv